OpenRA.Mods.Common/Traits/Repairable.cs

```diff
@@ -37,6 +35,9 @@ public class RepairableInfo : ITraitInfo, Requires<IHealthInfo>, Requires<IMoveI
 		[Desc("Boolean expression defining the condition under which the regular (non-force) enter cursor is disabled.")]
 		public readonly BooleanExpression RequireForceMoveCondition = null;
 
+		[Desc("At what distance to start repairing the actor.")]
```

Comment:
Imho we should add `from the center of the repairing actor` (otherwise it is unclear what `distance` is meant). Something like `Distance from the center of the repairing actor to start repairing this actor at.`

Reply:
Ok.